### DISCUSSION TITLE
Error_ early in processing for package loading despite I installed them and loaded them manually within the environment of NP3

### DISCUSSION THREAD

**Discussion by diosphenol** [danielatrivella/NP3_MS_Workflow#3] [General]
Hi Daniela and  the team. Congrats for the paper. I was tried to follow the tutorial but if gave the following upon processing execution:
ERROR

OUTPUT:Loading packages XCMS, MSnbase...

ERROR:Error: package or namespace load failed for 'MSnbase' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called 'scales'
Error: package 'MSnbase' could not be loaded
In addition: Warning message:
In fun(libname, pkgname) :
  mzR has been built against a different Rcpp version (1.0.2)
than is installed on your system (1.0.12). This might lead to errors
when loading mzR. If you encounter such issues, please send a report,
including the output of sessionInfo() to the Bioc support forum at 
https://support.bioconductor.org/. For details see also
https://github.com/sneumann/mzR/wiki/mzR-Rcpp-compiler-linker-issue.
Execution halted

I am using it on windows. Your help is appreciated.

**Comment by danielatrivella** [comment]
Dear Muhammad,


Thank you for your contact and interest on NP3 MS workflow.


Installation and running on windows are not yet straightforward - sorry for that.


You can try to reinstall your mzR and to set the environmental variable, as recommended here:

https://support.bioconductor.org/p/134630/#134631


However, we recommend Unix-based systems (Linux or iOS) in which installation and running of NP3 MS Workflow are ideal. Therefore you will be able to take the most from NP3 MS workflow - Step 6 and NP3 viewer, for example, are only currently available in Unix-based systems.


You can also try a Windows subsystem for Linux and install NP3 there.


Please let me know if any of these options worked for you.


Best wishes,

Daniela
—
Dra Daniela B B Trivella
Coordenadora de Descoberta de Fármacos
Laboratório Nacional de Biociências - LNBio
Centro Nacional de Pesquisa em Energia e Materiais – CNPEM
Rua Giuseppe Maximo Scolfaro, 10000<x-apple-data-detectors://0>
Campinas, SP Brasil<x-apple-data-detectors://0> 13083-100<tel:13083-100>
Fone: +55 19 3517-5055<tel:+55%2019%203517-5055>

Dr Daniela B B Trivella
Head of Drug Discovery
Brazilian Biosciences National Laboratory - LNBio
National Center for Research in Energy and Material  – CNPEM
10,000 Giuseppe Maximo Scolfaro St
Campinas, SP Brazil 13083-100<tel:13083-100>
Phone: +55 19 3517-5055<tel:+55%2019%203517-5055>

Aviso Legal: Esta mensagem e seus anexos podem conter informações
confidenciais e/ou de uso restrito. Observe atentamente seu conteúdo e
considere eventual consulta ao remetente antes de copiá-la, divulgá-la
ou distribuí-la. Se você recebeu esta mensagem por engano, por favor
avise o remetente e apague-a imediatamente.

Disclaimer: This email and its attachments may contain confidential
and/or privileged information. Observe its content carefully and
consider possible querying to the sender before copying, disclosing or
distributing it. If you have received this email by mistake, please
notify the sender and delete it immediately.

On 8 May 2024, at 20:56, Muhammad Alsherbiny ***@***.***> wrote:

﻿
Você não costuma receber emails de ***@***.*** Saiba por que isso é importante<https://aka.ms/LearnAboutSenderIdentification>

Hi Daniela and the team. Congrats for the paper. I was tried to follow the tutorial but if gave the following upon processing execution:
ERROR

OUTPUT:Loading packages XCMS, MSnbase...

ERROR:Error: package or namespace load failed for 'MSnbase' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
there is no package called 'scales'
Error: package 'MSnbase' could not be loaded
In addition: Warning message:
In fun(libname, pkgname) :
mzR has been built against a different Rcpp version (1.0.2)
than is installed on your system (1.0.12). This might lead to errors
when loading mzR. If you encounter such issues, please send a report,
including the output of sessionInfo() to the Bioc support forum at
https://support.bioconductor.org/. For details see also
https://github.com/sneumann/mzR/wiki/mzR-Rcpp-compiler-linker-issue.
Execution halted

I am using it on windows. Your help is appreciated.

**Reply by diosphenol** [reply]
Many Thanks I will try it over Linux to avoid windows hassle. really appreciate your help and congrats again.
